.github/workflows/nodejs-ci.yml

```diff
@@ -216,11 +236,29 @@ jobs:
         if: ${{ !cancelled() && !contains( inputs.excluded_jobs, 'build' )  }}
         run: |
           ${{ inputs.build_command }}
+      - name: Developer Tests
+        id: developer_tests
+        if: ${{ always() && inputs.run_dev_test && (github.event.pull_request.base.ref == 'qa' || contains(fromJson(inputs.developer_test_branch), github.event.pull_request.base.ref)) }}
```

Comment:
Only contains(fromJson(inputs.developer_test_branch), github.event.pull_request.base.ref)) is enough. `github.event.pull_request.base.ref == 'qa'` can be removed by setting the default to 'qa'.

Reply:
done